Jenkinsfile

```diff
@@ -54,8 +54,6 @@ pipeline {
             export LIQUIBASE_HOME=$WORKSPACE/rc
             export LIQUIBASE_WORKSPACE=$WORKSPACE/liquibase/changeLogs
 
-            chmod +x $WORKSPACE/liquibase/scripts/dbInit/z1_postgres_liquibase.sh
-            chmod +x $WORKSPACE/liquibase/scripts/dbInit/z2_rc_liquibase.sh
```

Comment:
The z1 file was marked as executable already.  The z2 file was not.  The empty change, below, adds +x to the z2 file.  Thus, we should no longer need the chmod in the Jenkinsfile.